core/trace/dt_span_serializer.go

```diff
@@ -254,16 +254,15 @@ func getSerializedResourceForSpanExport(spanResource *resource.Resource) ([]byte
 	if err != nil {
 		return nil, err
 	}
-	resource := protoResource.Resource{
```

Comment:
Why this renaming?

Reply:
We already import a package called `resource`. Technically allowed, but bad practice and can lead to confusing compiler errors.